package.json

```diff
@@ -33,7 +33,7 @@
     "test:e2e": "yarn e2e-tests test",
     "generate:test-services:v2": "yarn generate v2 && yarn lint:fix && yarn --frozen-lockfile",
     "generate:test-services:v4": "yarn generate v4 && yarn lint:fix && yarn --frozen-lockfile",
-    "generate:test-services:rest": "yarn rest-generator ts-node src/generator-cli.ts -i ../../test-resources/rest-service-specs -o ../../test-packages/test-services/rest && yarn lint:fix && yarn --frozen-lockfile",
+    "generate:test-services:rest": "yarn rest-generator sap-cloud-sdk-rest generate-rest-client -i ../../test-resources/rest-service-specs -o ../../test-packages/test-services/rest --clearOutputDir && yarn --frozen-lockfile",
```

Comment:
cloud you include the generation of the rest client also in the script generate-rest-service.ts so that we have it consistent.

Reply:
Also I see that the service has been generated and it is also used in the `rest.spec.ts` but I think a compilation of the generated sevice is missing to ensure syntactic correct content.

Reply:
I don't understand the first comment.
To the second: Yes, we should definitely do that in the generator directly. Currently we don't even generate a tsconfig.json that is why this is not there yet. I will add it with high priority to the todo list.

Reply:
as discussed per phone to the first point.